hp_search_benchmarks/nas_search_spaces/darts/cnn/startup-hook.sh

```diff
@@ -0,0 +1 @@
+git clone https://github.com/quark0/darts
```

Comment:
blocking: When referencing a git repository as a dependency, pin it to a commit hash so that the dependency doesn't change underneath this example and break it. 

non-blocking: if we do the thing where we fork and fix it up into a proper Python package, we can do a direct `pip install` from git here: https://pip.pypa.io/en/stable/reference/pip_install/#vcs-support